.gitignore

```diff
@@ -7,6 +7,8 @@ contracts/flattened
 .env
 .idea
 .vscode
+.old
+*.old
```

Comment:
Isn't it `.old` and `*.old` the same in this context?

Reply:
`*.old` also includes `a.old`, `b.old`, etc.

Reply:
I personally prefer to keep the gitignore as clean as possible, because you can set up a [global gitignore](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer) if you need to exclude certain files created by your editor.

Reply:
Yep, I agree. Let's delete them from the gitignore :P